board_report

```diff
@@ -11,81 +10,100 @@ source $OPENLAB_TOPDIR/Include/board_serial_op.sh
 source $OPENLAB_TOPDIR/Include/board_power_op.sh
 source $OPENLAB_TOPDIR/Include/pduop.sh
 
-board_all_no=`grep -E "^(BOARD)" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=BOARD)([^:]*)" | tr '\n' " "`
-board_all_usr=$(cut -d ":" -f1 $OPENLAB_CONF_DIR/$USER_INFO_FILE | grep ^[a-zA-Z]|tr '\n' " ")
+board_all_no=$(grep -E "^(BOARD)" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=BOARD)([^:]*)" | tr '\n' ' ')
+board_all_usr=$(cut -d ":" -f1 $OPENLAB_CONF_DIR/$USER_INFO_FILE | grep ^[a-zA-Z] | tr '\n' ' ')
 
 ################# Check the current usage of all the boards########################
-function busy()
+function status()
 {
     printf "%3s: %8s is used by -> %s\n" "Board No" "Board Type" "usr"
-    for value in ${board_all_no[*]}
+
+    for boardno in ${board_all_no[*]}
     do
-        usr=$(get_board_current_user $value)
-        board_type=`grep -E "^(BOARD${value}:).*" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=type=)([^ ,]*)"`
-        printf "%3s:  %8s is used by  -> %s\n"  "${value}" "${board_type}" "${usr}"
+        usr=$(get_board_current_user $boardno)
+        board_type=$(get_board_type $boardno)
+
+        if [ x"$usr" == x"" ]; then
+            usr="null"
+        fi
+
+        printf "%3s:  %8s is used by  -> %s\n" "${boardno}" "${board_type}" ${usr}
```

Comment:
%8s is not enough for 'overdrive', so overdrive line is not aligned with other lines.